### PR TITLE
Support Symfony5

### PR DESCRIPTION
**Pull request by larzuk91**
Symfony5 is released now and we all want to start using it :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":                       ">=5.3.3",
         "behat/behat":               "^3.4.3",
-        "symfony/framework-bundle":  "~2.0|~3.0|~4.0"
+        "symfony/framework-bundle":  "~2.0|~3.0|~4.0|~5.0"
```
Comment (AntoineLelaisant) [inline composer.json]: As Symfony versions `2.x` and `<3.4` are now out of support (https://symfony.com/releases) wouldn't it be better to drop their support now ?

```json
"require": {
    "symfony/framework-bundle": "~3.4|~4.2|~5.0"
}
```

File: .travis.yml [hunk 2]
```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPS=dev SYMFONY_VERSION='3.4.*'
     - php: 7.1
       env: DEVS=dev SYMFONY_VERSION='4.0.*'
+    - php: 7.2
```
Comment (theofidry) [inline .travis.yml]: I would suggest add a PHP 7.3 and 7.4 build as well

File: .travis.yml [hunk 3]
```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPS=dev SYMFONY_VERSION='3.4.*'
     - php: 7.1
```
Comment (theofidry) [inline .travis.yml]: Might be time to consider dropping 7.1 and below? They are all EOL

File: .travis.yml (hunk 2 above)
Comment (larzuk91) [reply .travis.yml]: Sounds logical to me, although, PHP 7.4 may generate some unexpected errors due it's a very recent version, but I'm just guessing

**Comment by piotrbrzezina** [comment]
any news about this PR ?

**Comment by olix21** [comment]
Can we help in any way this merge request?

**Comment by titomiguelcosta** [comment]
+1 for merging this PR

File: .travis.yml (hunk 3 above)
Comment (sroze) [reply .travis.yml]: Good idea, will do in another PR.

File: composer.json (hunk 1 above)
Comment (sroze) [reply composer.json]: Good idea, will do in another PR.

**Comment by matlar83** [comment]
+1! Super excited to start using Symfony5 soon :-)

**Comment by gonzalovilaseca** [comment]
Any news about merging this PR?

**Comment by adrienfr** [comment]
Hello, do you need any help to merge this PR and release a new tag?

**Comment by GPMathieu** [comment]
Hi, we need you merge this PR please :)

**Comment by larzuk91** [comment]
I don't have the rights to merge this PR, we need some from the maintainers of `behat/Symfony2Extension` to do it

**Comment by GPMathieu** [comment]
@stof @everzet ?

**Comment by chrisdejager** [comment]
There are issues when I try the above changes in my own fork. It doesn't result in a installable set of packages you are using symfony version 5. Try forcing symfony version 5 to see what I mean.

- behat/mink-browserkit-driver is not updated for sf5 yet, needs symfony/dom-crawler of a certain version which is too low for sf5.

**Comment by Nek-** [comment]
You should take a look at https://packagist.org/packages/friends-of-behat/symfony-extension

**Comment by wouter-veltman** [comment]
Can we get an update on this?

**Comment by sroze** [comment]
@everzet want to chip in? I don't have write permissions to this repo since you changed the organisation 😃

**Comment by ruudk** [comment]
Any news on this?

**Comment by yusefauto1** [comment]
24th August 2020, any news and update on this?

**Comment by larzuk91** [comment]
As @Nek- mentioned it. https://packagist.org/packages/friends-of-behat/symfony-extension is the way to go

**Comment by yusefauto1** [comment]
Hi @larzuk91 . thanks a lot for your attention and quick reply.  I saw that library. but really is disappointed instead of fixing refer to use another package. this is one of the bad things around the open-source world.

**Comment by ciaranmcnulty** [comment]
To make things clear and 'official', we are not going to be maintaining this extension going forwards.

The recommendation is to use the FriendsOfBehat one. In general we will be providing fewer extensions in the Behat/ namespace.

If migrating to FOB is not an option you can always fork this project into your own vendor

**Comment by larzuk91** [comment]
Hey @yusefauto1, yea I feel you man but the good part is it's not that hard to do the change. All I had to do (besides installing the library) was to change the config with this:
```
default:
  extensions:
    FriendsOfBehat\SymfonyExtension:
      bootstrap: path/to/your/bootstrap.php
```

I'll close the ticket since it won't be implemented.

**Comment by rpkamp** [comment]
> To make things clear and 'official', we are not going to be maintaining this extension going forwards.

It would be useful to put that in the README and point people to the FriendsOfBehat extension instead, to prevent wasting their time installing something that they'll have to replace later.

Also, it would be nice if this were indicated on the composer package as well.

**Pull request closed without merging**